core/templates/dev/head/pages/OppiaFooterDirective.ts

```diff
@@ -25,8 +25,14 @@ angular.module('oppia').directive('oppiaFooter', [
       bindToController: {},
       template: require('./oppia_footer_directive.html'),
       controllerAs: '$ctrl',
-      controller: [
-        function() {}
+      controller: ['SITE_FEEDBACK_FORM_URL',
+        function(SITE_FEEDBACK_FORM_URL) {
+          var ctrl = this;
+
+          ctrl.$onInit = function() {
+            ctrl.siteFeedbackFormUrl = SITE_FEEDBACK_FORM_URL;
+          };
```

Comment:
I don't think this needs to be in the `$onInit`, but I'm not sure. Please test it without it.

Reply:
done